### DISCUSSION TITLE
Multi selection on relationship field.

### DISCUSSION THREAD

**Discussion by emreincu** [wpengine/atlas-content-modeler#586] [Ideas]
Hi,

first of all, thanks for your all efforts. I have tested the Atlas Content Modeler and I think you can change the relationship field type as a multi-selection box instead of a select box.

Let's say we have three different content models, "Page", "Banner" and "Article" etc. And let's say I want to add a relationship named "components" field type into the "Page" content model. And I want to select Banners and Articles in the same field. 

Currently, we can not do this feature cause we can select only one content model type.

Thank you.

**Comment by nickcernis** [comment]
Thank you for this feature request, @emreincu!

We'll leave this open to gauge demand, and to let you know if we start work on the feature.
